www/js/index.js

```diff
@@ -21,9 +23,11 @@ function escKey() {
 }
 
 function waheguruSlide() {
+  // waheguru slide shortcut
   global.controller.sendText('vwihgurU', true);
 }
 function ikOankarSlide() {
```

Comment:
Can we make sure this is renamed Mool Mantra globally instead of IkOankarSlide

Reply:
👍